tests/contrib/flask/test_request.py

```diff
@@ -556,9 +555,24 @@ def fivehundred():
                 "flask.handle_exception",
                 "flask.do_teardown_request",
                 "flask.do_teardown_appcontext",
-            ],
-            [s.name for s in spans],
-        )
+            ]
+        else:
+            # Flask >= 1.1
+            span_names = [
+                "flask.request",
+                "flask.try_trigger_before_first_request_functions",
+                "flask.preprocess_request",
+                "flask.dispatch_request",
+                "tests.contrib.flask.test_request.fivehundred",
+                "flask.handle_user_exception",
+                "flask.handle_exception",
+                "flask.process_response",
```

Comment:
sneaky, sneaky


would it read more clear as something like:


```python
            span_names = [
                "flask.request",
                "flask.try_trigger_before_first_request_functions",
                "flask.preprocess_request",
                "flask.dispatch_request",
                "tests.contrib.flask.test_request.fivehundred",
                "flask.handle_user_exception",
            ]
            if FLASK_VERSION_TUPLE >= (1, 1, 0):
                span_names.append("flask.process_response")
            span_names += [
                "flask.do_teardown_request",
                "flask.do_teardown_appcontext",
            ]
```

Not a strong opinion, just thinking if there is a way to make it easier to spot what is different for Flask  >= 1.1.0

Reply:
Sure why not :)